microcloud/test/includes/microcloud.sh

```diff
@@ -934,12 +934,12 @@ setup_system() {
     # Snaps can occasionally fail to install properly, so repeatedly try.
     lxc exec "${name}" -- sh -c "
       while ! test -e /snap/bin/microceph ; do
-        snap install microceph || true
+        snap install microceph --cohort='+' || true
```

Comment:
should we use `--edge` or at least `<LTS track>/edge`? for this and microovn to catch regressions early??

Reply:
We should probably have both. We should run tests on `stable` because that's what we know today will definitely work, and we should have some cases for `edge` to ensure we anticipate any upcoming breakages.

Reply:
yeah agreed, with a preference towards edge builds initially.